code/modules/supermatter/supermatter.dm

```diff
@@ -453,7 +454,7 @@
 		"power" = power,
 		"sigtype" = "status"
 	)
-	radio_connection.post_signal(src, signal)
+	radio_connection.post_signal(src, signal, RADIO_SUPERMATTER, SS_INIT_RUST)
```

Comment:
what? why are you using a define for the rust's subsystem's initialization order as the range parameter?

Reply:
that's radius 26, which felt like a good number

Reply:
ok but this is abuse and i'm calling the cops
make it its own define